src/Database/Options/WorkloadManagerDatabaseSettings.cs

```diff
@@ -0,0 +1,16 @@
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+namespace Monai.Deploy.WorkflowManager.Database.Options
+{
+    public class WorkloadManagerDatabaseSettings
```

Comment:
Are these settins coming from the `appsettings.json` file? If so, may need to decorate with `ConfigurationKeyName`

Reply:
Yeah they are, have added the decorator now.